AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreLocalUsageTests.swift

```diff
@@ -0,0 +1,289 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSMobileClient
+import AWSPluginsCore
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreLocalUsageTests: LocalUsageIntegrationTestBase {
+
+    func testOneSort() throws {
+        _ = saveData(num: 20)
+
+        let queryFinished = expectation(description: "Query post completed")
```

Comment:
DataStoreConfigurationTests uses `saveSuccess`, i think that works for this as well since it's fulfilled when the query was successful, so like `let querySuccess`

Reply:
renamed